gn3/api/correlation.py

```diff
@@ -39,13 +57,17 @@ def compute_lit_corr(species=None, gene_id=None):
     are fetched from the database this is the only case where the db\
     might be needed for actual computing of the correlation results"""
 
-    database_instance = mock.Mock()
+    # database_instance = mock.Mock()
+    database_instance, _cursor_object = database_connector()
     target_traits_gene_ids = request.get_json()
+    target_trait_gene_list = list(target_traits_gene_ids.items())
 
     lit_corr_results = compute_all_lit_correlation(
-        database_instance=database_instance, trait_lists=target_traits_gene_ids,
+        database_instance=database_instance, trait_lists=target_trait_gene_list,
         species=species, gene_id=gene_id)
 
+    database_instance.close()
```

Comment:
*nitpick*: Best to name "database_instance" simply "conn". My rationale is that, "conn" in most code bases is what's normally used.

PS: Naming is hard!